civet.dev/cheatsheet.md

```diff
@@ -18,25 +18,114 @@ The TypeScript code on
 
 [[toc]]
 
-## Basics
+## TC39 Left a Lot on the Table
```

Comment:
This heading seems a little ... controversial? I wonder if it would be better to be positive with something like "Making JavaScript More Convenient"?

Reply:
Controversy can sometimes be good for attention :)